### PR TITLE
README Spanish translation

### PR DESCRIPTION
**Pull request by orlybg**
Created README Spanish translation

### CODE REVIEW

File: README.es.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+# Codigo Fuente de MS-DOS v1.25 y v2.0 
+Este repositorio contiene el código fuente original y binarios compilados para MS-DOS v1.25 y MS-DOS v2.0.
+
+Estos son los mismos archivos [originalmente compartidos al Museo Histórico de Ordenadores en Marzo 25 de 2014]( http://www.computerhistory.org/atchm/microsoft-ms-dos-early-source-code/) y están siendo (re)publicados en este repositorio para que puedan ser encontrados mas fácilmente, referenciados en escritos y trabajos externos, y permitir la exploración y experimentación para aquellos interesados en sistemas operativos para PC tempranos.
+
+# Licencia 
+Todos los archivos contenidos en este repositorio fueron liberados bajo la [Licencia MIT (OSI)](https://es.wikipedia.org/wiki/Licencia_MIT) según el [archivo de Licencia](https://github.com/Microsoft/MS-DOS/blob/master/LICENSE.md) almacenado en la raíz de este repositorio.
+
+# Contribuye!
+Los archivos fuente en este repositorio son para referencia histórica y permanecerán estáticos, así que por favor no hagas *pull requests* sugiriendo modificación alguna a los archivos de código fuente, pero sientete libre de bifurcar *fork* y experimentar 😊.
```
Comment (bitcrazed) [inline README.es.md]: Please avoid markdown formatting for punctuation. If you intended to wrap in asterisks, use `\*` instead.

File: README.es.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+# Licencia 
+Todos los archivos contenidos en este repositorio fueron liberados bajo la [Licencia MIT (OSI)](https://es.wikipedia.org/wiki/Licencia_MIT) según el [archivo de Licencia](https://github.com/Microsoft/MS-DOS/blob/master/LICENSE.md) almacenado en la raíz de este repositorio.
+
+# Contribuye!
+Los archivos fuente en este repositorio son para referencia histórica y permanecerán estáticos, así que por favor no hagas *pull requests* sugiriendo modificación alguna a los archivos de código fuente, pero sientete libre de bifurcar *fork* y experimentar 😊. 
+
+Sin embargo, si desea enviar contenido adicional que no sea de origen o modificaciones a archivos que no sean de fuente (por ejemplo, este archivo README), envíe una petición *PR* y lo revisaremos y consideraremos.
```
Comment (bitcrazed) [inline README.es.md]: Same as above, re. `*PR*`

**Comment by orlybg** [comment]
@bitcrazed markdown removed, used parenthesis instead for the standard English terms for fork and PR

File: README.es.md (hunk 1 above)
Comment (orlybg) [reply README.es.md]: @bitcrazed markdown removed, used parenthesis instead for the standard English terms for fork and PR to add clarity to the not so commonly used Spanish translations.

**Comment by bitcrazed** [comment]
Many thanks. Merged.

**Pull request merged**